data/aws_cluster.json

```diff
@@ -1,11 +1,10 @@
 {
   "num_workers": 1,
-  "cluster_name": "API_Metastore_Work_Leave_Me_Alone",
+  "cluster_name": "Workspace_Migration_Work_Leave_Me_Alone",
   "spark_version": "6.5.x-scala2.11",
   "aws_attributes": {
       "first_on_demand": 1,
       "availability": "SPOT_WITH_FALLBACK",
-      "zone_id": "us-west-2b",
```

Comment:
we should just not specify the zone_id so that this file can be used region agnostically.

Reply:
We should use `auto` so it chooses a zone_id automatically in the future. By default Databricks will fix it to one AZ / zone_id. `auto` is a better option that we can improve upon later.

Reply:
thank you! doesn’t databricks still randomly pick a zone_id? (or are you saying it’s hard-coded in terms of which zone id it will be used?) Hopefully, Databricks pick an available zone id though?

Reply:
No, we've never randomly set the `zone_id`. We have a concept of `default`. If you go to any cluster UI, the defaults always set. If the API does not choose a zone, it will choose the default. The `auto` intelligently chooses the `zone_id` by looking at available ip space and if the instance isn't available, it can fallback to another zone.